app/main/helpers/search_summary_manifest.yml

```diff
@@ -85,6 +85,28 @@
   filtersPreposition:
   filterRules:
   conjunction: and
+
+
+-
+  id: Supplier type
+  labelPreposition: of type
```

Comment:
I'm going back through the filter labels I've done to correct capitalisation.

We'll try to get some of Roz's time tomorrow to discuss best solution for prepositions etc for MVP.